assets/model_monitoring/components/src/model_monitor_compute_histogram/run.py

```diff
@@ -35,13 +40,16 @@ def run():
     parser.add_argument("--histogram", type=str)
     args = parser.parse_args()
 
-    df = read_mltable_in_spark(args.input_data)
+    df = try_read_mltable_in_spark_with_warning(args.input_data, "input_data")
+
+    if not df:
+        print("Skipping histogram generation.")
+        return
 
-    histogram_buckets = None
-    try:
-        histogram_buckets = read_mltable_in_spark(args.histogram_buckets)
-    except Exception:
-        print("No histogram buckets detected.")
+    histogram_buckets = try_read_mltable_in_spark(
+        args.histogram_buckets, "histogram_buckets"
+    )
+    if not histogram_buckets:
```

Comment:
I see you removed a line of logging for no histogram. Is this intented?

Reply:
I'll add back!